emission/storage/decorations/token_queries.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+# Utility class to return useful user queries
+from future import standard_library
+standard_library.install_aliases()
+from builtins import *
```

Comment:
```suggestion
```

These are from python2 -> python3 upgrade; we can just remove this since we are writing directly in python3 now.